### PR TITLE
Add option to helm chart for "global mode"

### PR DESCRIPTION
**Pull request by funkypenguin**
Hey guys!

Another PR - this one adds the option to the helm chart to enable "global mode", effectively removing the `WATCH_NAMESPACE` var, and turning the Role/Rolebinding into a ClusterRole/ClusterRolebinding.

I've tested on my cluster, and the RBAC components are working as expected 👍🏻 

D

### CODE REVIEW

File: helm/fdb-operator/values.yaml [hunk 1]
```diff
@@ -4,3 +4,5 @@ operator:
   tag: 0.24.0
   role: fdb-kubernetes-operator-manager-role
   rolebinding: fdb-kubernetes-operator-manager-rolebinding
+  globalMode:
+    enabled: false
```
Comment (johscheuer) [inline helm/fdb-operator/values.yaml:8]: Missing new line

File: helm/fdb-operator/templates/rbac/rbac_role_binding.yaml [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
+  {{- else }}
   kind: Role
+  {{- end }}
   name: {{ .Values.operator.role }}
 subjects:
 - kind: ServiceAccount
   name: fdb-kubernetes-operator-controller-manager
+  {{- if .Values.operator.globalMode.enabled }}
+  namespace: {{ .Release.Namespace }}
+  {{- end }}
```
Comment (johscheuer) [inline helm/fdb-operator/templates/rbac/rbac_role_binding.yaml:24]: Missing new line

File: helm/fdb-operator/templates/rbac/rbac_role_binding.yaml [hunk 3]
```diff
@@ -1,13 +1,24 @@
...
+  {{- if .Values.operator.globalMode.enabled }}
+  kind: ClusterRole
+  {{- else }}
   kind: Role
+  {{- end }}
   name: {{ .Values.operator.role }}
 subjects:
 - kind: ServiceAccount
   name: fdb-kubernetes-operator-controller-manager
+  {{- if .Values.operator.globalMode.enabled }}
```
Comment (johscheuer) [inline helm/fdb-operator/templates/rbac/rbac_role_binding.yaml:22]: I think we can put this here as a default and remove the if statement

**Review by johscheuer — CHANGES_REQUESTED** [review]

File: helm/fdb-operator/values.yaml (hunk 1 above)
Comment (funkypenguin) [reply helm/fdb-operator/values.yaml]: 👍🏻

File: helm/fdb-operator/templates/rbac/rbac_role_binding.yaml (hunk 3 above)
Comment (funkypenguin) [reply helm/fdb-operator/templates/rbac/rbac_role_binding.yaml]: Done!

**Review by johscheuer — APPROVED** [review]
👍

**Pull request merged**